site-manage/site-manage-tool/tool/src/webapp/vm/sitebrowser/chef_sitebrowser_simpleSearch.vm

```diff
@@ -32,7 +33,7 @@
 					<div class="radio indnt3">
 						<input type="radio" name="siteType" class="siteTypeTerm" id="siteType$siteTypeCount" value="$type" />
 						<label for="siteType$siteTypeCount">$type $tlang.getString("se.sites")</label>
-
+						<label for="selectTerm">Select a Term:</label>
```

Comment:
this should be a translated string?

Reply:
I realized that the string "Select a Term:" is not needed as there is text already present on the element and JAWS is repeating the instructions for that element. I will update it such that no text is there within the label tag to avoid redundancy.